scalastyle-test-config.xml

```diff
@@ -30,7 +30,7 @@
     <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"></check>
     <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
         <parameters>
-            <parameter name="maxLineLength"><![CDATA[200]]></parameter>
```

Comment:
is it good to change this? should we keep it low to discourage people doing long lines of code?

Reply:
In production this is still low. We have changed it once with Ben to 200, so I thought I could increase it again, as this is for tests only. We should address the warnings one way or another. We can check with Ben or someone else to see what we should do.

Reply:
I didn't mind the increase to 200 but I agree with Joey we should still keep it relatively similar to the production file so the tests stay rather readable.

Reply:
I'm with joey on this one. I don't see much point in raising it higher.

Reply:
Sure, I will put it back to 200. We have to resolve the warnings though. Probably will be part of another work.